<launch>

	<param name = "gridRows" value = "5" />
	<param name = "gridCols" value = "5" />
	<param name = "numBlocks" value = "3" />
	<param name = "blockLocaleRow" value = "3" />
	<param name = "blockLocaleCol" value = "3" />
	<param name = "configuration" value = "stacked_ascending" />
	<param name = "goalState" value = "stacked_descending" />
	<param name = "isOneArmSolution" value = "True" />
	
	<node name = "robot_interface" pkg = "gabe_ricky_sarah_proj1" type = "robot_interface.py" />
	<node name = "controller" pkg = "gabe_ricky_sarah_proj1" type = "controller.py" />
</launch>
